scripts/new_project.kts

```diff
@@ -248,7 +248,7 @@ object NewProject {
             "$projectPath${fileSeparator}gradlew",
             "-p",
             "$projectPath",
-            "testStagingDebugUnitTest"
+            "jacocoTestReport"
```

Comment:
I have mixed feeling of this solution 🤔 what if we don't use jacoco anymore, given that there is Kover on the corner waiting for the evaluation 🤔

Reply:
`testStagingDebugUnitTest` is only running tests for `app` module, hence the update

If we end up using Kover, we can always adapt it again later 🚀

Reply:
have you tried just `test` or `testStagingDebug`? I think jacoco must run a similar gradle command under the hood, so probably you can sneak into the gradle log to see what has been called. Running jacocoTestReport is slower and you are not going to read the report at the time you init the project right?

Reply:
@sleepylee You're right, `jacocoTestReport` should be avoided, as we won't be reading the report 👍 

Unfortunately, I can't make it work with other commands:
- `test` : Executes tests for both `debug` & `release` variant too, which should be avoided
- `testStagingDebug` : Only runs tests for `app` module
- `testDebugUnitTest` : Only runs tests for `data` module

Maybe something is wrong with our set-up 💭 

<img width="659" alt="Screen Shot 2565-08-18 at 12 32 18" src="https://user-images.githubusercontent.com/18277915/185301570-968f324e-95bc-4a7c-99f6-6327b809f411.png">

Reply:
@Tuubz seems like it's possibly because we don't have `staging` flavor in `data`, so perhaps we can execute 2 commands:
```
testStagingDebug -> this ensure any module that has Staging flavor will run unit test
testDebugUnitTest -> any module that does not have extra flavor, must run the debug build type
```

pros: 
 - you don't need to think about this again when moving to Kover or any other coverage report tools, or even if we remove it entirely.
cons:
 - 🤔

Reply:
@sleepylee thanks for the feedback, I'll go with your suggestion.

I think we'll end up needing to execute 3 commands (`app`, `data` & `domain`)

Reply:
🤔  @Tuubz correct me if I'm wrong but doesn't `testDebugUnitTest` execute `:data` AND `:domain`?

Reply:
@sleepylee No, it only runs for `:data` 😞